source/docs/transportation/alt-fuel-stations-v1/get.html.md.erb

```diff
@@ -153,175 +157,225 @@ The response is composed of the fuel station matching the requested query. The s
-    "ng_vehicle_class": null,
-    "access_days_time_fr": null,
-    "intersection_directions_fr": null,
-    "bd_blends_fr": null,
-    "groups_with_access_code_fr": "Privé",
-    "ev_pricing_fr": null,
-    "federal_agency": {
-      "id": 8,
-      "code": "DOE",
-      "name": "U.S. Department of Energy"
+  "features": [
+    {
+      "type": "Feature",
+      "geometry": {
+        "type": "Point",
+        "coordinates": [
+          -105.156235,
+          39.740932
+        ]
+      },
+      "properties": {
+        "access_code": "public",
+        "access_days_time": "24 hours daily",
+        "access_detail_code": null,
+        "cards_accepted": null,
+        "date_last_confirmed": "2022-06-13",
+        "expected_date": null,
+        "fuel_type_code": "ELEC",
+        "groups_with_access_code": "Public",
+        "id": 194116,
+        "open_date": "2021-08-19",
+        "owner_type_code": null,
+        "status_code": "E",
+        "restricted_access": null,
+        "station_name": "1687 COLE BLVD EAST STATION 1",
+        "station_phone": "888-758-4389",
+        "updated_at": "2022-06-13T01:40:48Z",
+        "facility_type": null,
+        "geocode_status": "GPS",
+        "city": "Lakewood",
+        "intersection_directions": null,
+        "plus4": null,
+        "state": "CO",
+        "street_address": "1687 Cole Blvd",
+        "zip": "80401",
+        "country": "US",
+        "bd_blends": null,
+        "cng_dispenser_num": null,
+        "cng_fill_type_code": null,
+        "cng_psi": null,
+        "cng_renewable_source": null,
+        "cng_total_compression": null,
+        "cng_total_storage": null,
+        "cng_vehicle_class": null,
+        "e85_blender_pump": null,
+        "e85_other_ethanol_blends": null,
+        "ev_connector_types": [
+          "J1772"
+        ],
+        "ev_dc_fast_num": null,
+        "ev_level1_evse_num": null,
+        "ev_level2_evse_num": 2,
+        "ev_network": "ChargePoint Network",
+        "ev_network_web": "http://www.chargepoint.com/",
+        "ev_other_evse": null,
+        "ev_pricing": null,
+        "ev_renewable_source": null,
+        "hy_is_retail": null,
+        "hy_pressures": null,
+        "hy_standards": null,
+        "hy_status_link": null,
+        "lng_renewable_source": null,
+        "lng_vehicle_class": null,
+        "lpg_primary": null,
+        "lpg_nozzle_types": null,
+        "ng_fill_type_code": null,
+        "ng_psi": null,
+        "ng_vehicle_class": null,
+        "access_days_time_fr": null,
+        "intersection_directions_fr": null,
+        "bd_blends_fr": null,
+        "groups_with_access_code_fr": "Public",
+        "ev_pricing_fr": null,
+        "ev_network_ids": {
+          "station": [
+            "USCPIL9075981"
+          ],
+          "posts": [
+            "16005951",
+            "16005961"
+          ]
+        },
+        "distance": 0.25756,
+        "distance_km": 0.4145
+      }
     }
-  }
+  ]
```

Comment:
This appears to be the GeoJSON output from the "all" endpoint, not the "show" endpoint, since it's returning an array instead of a single item.

Reply:
fixed